src/visualization/utils/getFormatter.test.ts

```diff
@@ -1,5 +1,5 @@
 import {getFormatter} from './getFormatter'
-import {DEFAULT_TIME_FORMAT} from '../../shared/constants'
+import {DEFAULT_TIME_FORMAT} from '../../utils/datetime/constants'
```

Comment:
relative paths were kept as relative, just to avoid any weird behavior like yesterday.